darwin/datatypes.py

```diff
@@ -32,6 +32,37 @@ def get_sub(self, annotation_type: str) -> Optional[SubAnnotation]:
                 return sub
 
 
+@dataclass(frozen=True, eq=True)
+class VideoAnnotation:
+    annotation_class: AnnotationClass
+    frames: Annotation
+    keyframes: List[bool]
+    segments: List[List[int]]
+    interpolated: bool
+
+    def get_frame(self, frame_index):
+        return frames[frame_index]
```

Comment:
It would be awesome if we could annotate types here, too.
It's also useful for review and documentation.

For example: is `frame_index` an integer or a string? I guess a string, but I would have to check that we are consistent every time we call this function to be 100% sure.

Reply:
Annotated this function too. `frame_index` are ints but I'd also need to check that we are consistent with that.